pallets/subtensor/src/staking/set_children.rs

```diff
@@ -1,26 +1,25 @@
 use super::*;
+use sp_core::Get;
```

Comment:
Is there a test for the new rate limit mechanic?

Reply:
Good point! I just added `test_do_set_child_cooldown_period` to test that childkeys do not apply immediately. 

The rate limits though stayed unchanged and it still applies to `set_children` extrinsic. The test `test_set_children_rate_limit_fail_then_succeed` works as before. The new tests that verify the logic are:

```
test_do_set_child_below_min_stake
test_do_remove_stake_clears_pending_childkeys
test_set_children_rate_limit_fail_then_succeed
test_do_set_child_cooldown_period
```